Automattic/wp-calypso
# Site Settings: Move stylesheets into dedicated bundle

This pull request fixes https://github.com/Automattic/wp-calypso/issues/18488 by reverting https://github.com/Automattic/wp-calypso/pull/18477/ and taking a different approach which consists in extracting the stylesheets used by the `Site Settings` section from the main CSS bundle, and moving them into a dedicated bundle that is only loaded when the section is accessed (thanks to the new CSS splitting feature introduced in https://github.com/Automattic/wp-calypso/pull/17988). This allows loading the media modal styles only for this specific section as well as the `Post Editor` section. Finally, this pull request makes sure only the necessary stylesheets, variables, and mixins are imported from all the new CSS section bundles.

#### Testing instructions

1. Run `git checkout update/css-bundles` and start your server, or open a [live branch](https://calypso.live/?branch=update/css-bundles)
2. Open the [`Signup` page](http://calypso.localhost:3000/start)
3. Assert that this section displays as before
4. Open the [`Login` page](http://calypso.localhost:3000/log-in)
5. Assert that this page displays as before
6. Open the [`General Settings` page](http://calypso.localhost:3000/settings/general)
7. Assert that the page (and any other subsection) displays as before
8. Click the `Change` button in the `Site Profile` section
9. Asserts that the media library displays normally
10. Click the `Write` button in the top bar to access the post editor
11. Click the cross icon to open the media library
12. Click any image and assert that the `Edit`  and trash buttons are correctly aligned

You may want to check these steps again by toggling the mobile device view in your browser's devtools.

#### Reviews
